### PR TITLE
Combine amp-analytics var docs with var substitutions doc re: #1302

### PR DESCRIPTION
**Pull request by bpaduch**
@rudygalfi & @avimehta  - I've taken Avi's work and Rudy's suggestion to create a combined list of variable substitutions.  (re: #1302, [Avi's PR](https://github.com/ampproject/amphtml/pull/5525)).

A couple things to note:
1. I didn't include the [amp-access-analytics variables](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access-analytics.md#access-analytics-variables) as they seemed to be a specialized case.  Should they be included in this list?  
2. I included more variables than were described in the original docs, essentially I checked against the vars "list" in [vendors.js](https://github.com/ampproject/amphtml/blob/e7111b25bdd03b091c0e510ded619588b3fee4c8/extensions/amp-analytics/0.1/vendors.js). Is that okay?  Essentially, some vars were described for amp-analytics but not in the var-substitution doc (e.g.,  `scrollHeight`, `scroll...`)

### CODE REVIEW

**Comment by avimehta** [comment]
Have not gone through the complete list yet but a few things: 

- I'll send out a PR to add the `not supported` variables to `amp-analytics`. Feel free to update this PR. 
- The amp-analytics variables are used by enclosing them in `${}` so `RANDOM` will looks like `${random}`. It will be nice to use the full format in the `amp-analytics` variable name.

File: spec/amp-var-substitutions.md [hunk 1]
```diff
@@ -32,422 +29,868 @@ will still update to the most current values of the variables.  Note that some
 variables will not allow for expansion for A4A creatives.  Specifics will be
 addressed at a later time.
 
-## Page and Content
+## Variables 
+
+The tables below list the available URL variables grouped by type of usage. Further down in this document, are [descriptions](#variable-descriptions) of each of the variables, along with example usages.
+
+* [Page and Content](#page-and-content)
```
Comment (avimehta) [inline spec/amp-var-substitutions.md]: Maybe this section is not needed?

File: spec/amp-var-substitutions.md [hunk 2]
```diff
@@ -32,422 +29,868 @@ will still update to the most current values of the variables.  Note that some
...
+* **platform variable**: `PAGE_LOAD_TIME`
+  *  Example: <br>
+  ```html
+  <amp-pixel src="https://foo.com/pixel?pageLoadTime=PAGE_LOAD_TIME"></amp-pixel>
+  ```
+* **amp-analytics variable**: `${pageLoadTime}`
+  * Example value: `220`
 
-### PAGE_DOWNLOAD_TIME
+### Page View ID
```
Comment (avimehta) [inline spec/amp-var-substitutions.md]: I think this needs to be ####.

File: spec/amp-var-substitutions.md [hunk 3]
```diff
@@ -32,422 +29,868 @@ will still update to the most current values of the variables.  Note that some
...
-may make a request to something like `https://foo.com/pixel?host=pinterest.com:9000`.
+<!-- cid is not provided until `user-consent` is dismissed -->
+<amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(cid-scope-cookie-fallback-name,user-consent-id)"></amp-pixel>
+  ```
+* **amp-analytics variable**: `${clientId}`
+  * Example usage: `${clientId(foo)}`
+  * Example value: `U6XEpUs3yaeQyR2DKATQH1pTZ6kg140fvuLbtl5nynbUWtIodJxP5TEIYBic4qcV`
 
-### SOURCE_HOSTNAME
+**Arguments**
```
Comment (avimehta) [inline spec/amp-var-substitutions.md]: In the rendered doc, the 'arguments' and client id look exactly the same and it looks like arguments is just another type of variable. If there any way to change the formatting?

File: spec/amp-var-substitutions.md (hunk 1 above)
Comment (avimehta) [inline spec/amp-var-substitutions.md]: my thinking was along the lines that this results in 2 levels of tabels to get to the variables. Given that this is a small table and the links from here go directly below, maybe this is not needed.

But, I am okay with whatever you decide here and will go with that.

Comment (bpaduch) [inline spec/amp-var-substitutions.md]: I see that but for consistency, I'll leave it.  It's possible too that someone many change the order, so it's good to keep it.

**Comment by bpaduch** [comment]
@rudygalfi  - let me know if it's good to go.  Ideally, I'd like to get this out before any more vars are added to the analytics-vars doc :)

**Comment by rudygalfi** [comment]
This is a dream come true. Looks fantastic!

:shipit:

**Pull request merged**